pkgs/development/python-modules/credstash/default.nix

```
@@ -17,7 +17,7 @@ buildPythonPackage rec {
   # file ensures that Python imports the module from site-packages library.
   postInstall = "rm $out/bin/credstash.py";
 
-  nativeBuildInputs = [ nose ];
+  nativeBuildInputs = [ pytest ];
```

this should really be checkInputs
```suggestion
  checkInputs = [ pytest ];
```

---

you also need to list this as an input to the nix expression:
```
undefined variable 'pytest' at /var/lib/ofborg/checkout/repo/38dca4e3aa6bca43ea96d2fcc04e8229/mr-est/packet-spot-eval-3/pkgs/development/python-modules/credstash/default.nix:20:25
```

---

Should be good now. Strange it worked locally oO